partner_contact_nutrition_diet/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================
+Partner Contact Nutrition Diet
+==============================
+
+This module adds the nutrition diet to a partner contact:
+
+* Lifestyle
+* Carnivore
+* Pescatarian
+* Vegetarian
+* Vegan
+
+Usage
+=====
+
+* Go to Contacts
+* Create or select a contact
+* Go to the Nutrition tab to select their diet
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/134/10.0
+
+
+Known issues / Roadmap
```

Comment:
@lasley remove section if not content

Reply:
He is no longer working with Odoo, so this PR should be superseeded if you want it to be finished.

Reply:
@pedrobaeza Maybe we should close all the nutrition PR's then, as they will probably not be maintained?

Reply:
What you prefer.